tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>typo: regression, not regrression</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuromatchAcademy/course-content/pull/104/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thanks. There's a few other little things, I'm looking through.

Reply:
For small things like this we should probably note and handle during post pre pod revisions.

Reply:
Generally agree, though this is in the tutorial header in big font, so it's very obvious

Reply:
New links are up.